src/composite/Button/__tests__/Button.test.tsx

```diff
@@ -0,0 +1,88 @@
+import * as React from "react";
+import { fireEvent, render, screen } from "@testing-library/react";
+import { composeStory } from "@storybook/testing-react";
+import Meta, {
+  Primary as PrimaryStory,
+  Secondary as SecondaryStory,
+  Link as LinkStory,
+} from "../Button.stories";
+import { defaultStyles, sizes, variants } from "../constants";
+import { SIZES, VARIANTS } from "../model";
+
+const getClassName = (variant: VARIANTS, size: SIZES) => {
+  const defaultStylesAsClassNames = defaultStyles;
+  const variantClassNames = variants[variant];
+  const sizeClassNames = sizes[size];
+
+  return `${defaultStylesAsClassNames} ${variantClassNames} ${sizeClassNames}`;
+};
+
+describe("Primary Button", () => {
+  const Primary = composeStory(PrimaryStory, Meta);
+
+  it("renders with default args", () => {
```

Comment:
These are basic tests, we will need to improve on it, its fine for now, we will do this later